src/components/ProductList.jsx

```
@@ -19,14 +20,35 @@ function ProductList() {
 
   const handleDelete = async (id) => {
     // TODO: DELETE API를 호출하고 fetchProducts() 호출
+    try {
+      const apiResponse = await instance.delete(id);
```

ESLint

[error] 'apiResponse' is assigned a value but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)